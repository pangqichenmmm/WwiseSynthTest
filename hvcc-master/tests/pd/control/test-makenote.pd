#N canvas 1054 287 490 514 10;
#X obj 142 275 unpack f f;
#X obj 144 402 pack f f;
#X msg 188 166 60 124;
#X msg 283 101 60;
#X msg 212 144 70 -1203;
#X obj 141 48 loadbang;
#X obj 141 70 t b b b b b;
#X obj 144 363 makenote 60 50;
#X msg 234 111 10 0;
#X obj 122 157 t b b;
#X msg 241 273 125;
#X msg 121 215 55 100;
#X obj 144 424 print;
#X obj 357 337 sig~ 0;
#X obj 357 359 dac~;
#X connect 0 0 7 0;
#X connect 0 1 7 1;
#X connect 1 0 12 0;
#X connect 2 0 0 0;
#X connect 3 0 7 0;
#X connect 4 0 0 0;
#X connect 5 0 6 0;
#X connect 6 0 9 0;
#X connect 6 1 2 0;
#X connect 6 2 4 0;
#X connect 6 3 8 0;
#X connect 6 4 3 0;
#X connect 7 0 1 0;
#X connect 7 1 1 1;
#X connect 8 0 0 0;
#X connect 9 0 11 0;
#X connect 9 1 10 0;
#X connect 10 0 7 2;
#X connect 11 0 0 0;
#X connect 13 0 14 0;
