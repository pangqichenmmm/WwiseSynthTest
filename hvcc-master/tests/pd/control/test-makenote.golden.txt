[@ 0.000] print: 60 60
[@ 0.000] print: 70 -1203
[@ 0.000] print: 60 124
[@ 0.000] print: 55 100
[@ 50.000] print: 60 0
[@ 50.000] print: 70 0
[@ 50.000] print: 60 0
[@ 125.000] print: 55 0